blocks/search-results/search-results.js

```diff
@@ -139,9 +140,69 @@ async function searchPages(placeholders, term, page) {
 
   const totalResults = result.length;
   const totalPages = Math.ceil(totalResults / resultsPerPage);
-
   addPagingWidget(div, page, totalPages);
-
+  const paginationblock = div.querySelector('ul');
+  const paginationLimit = 5;
+  if (totalPages > paginationLimit) {
+    let elementForward = 0;
+    const threeDotsAfter = document.createElement('li');
+    const ata = document.createElement('a');
+    ata.innerText = '...';
+    threeDotsAfter.appendChild(ata);
+
+    const threeDotsBefore = document.createElement('li');
+    const atb = document.createElement('a');
+    atb.innerText = '...';
+    threeDotsBefore.appendChild(atb);
+
+    const firstElement = paginationblock.querySelector('.prev.page').nextElementSibling;
+    const lastElement = paginationblock.querySelector('.next.page').previousElementSibling;
+    firstElement.after(threeDotsBefore);
+    lastElement.before(threeDotsAfter);
+
+    if (page < (paginationLimit - 1)) {
+      firstElement.nextElementSibling.classList.add('notvisible');
+      const currentElement = paginationblock.querySelector('.active');
+      // if (page === 0) {
+      //   elementForward = currentElement.nextElementSibling.nextElementSibling.nextElementSibling;
+      // } else {
+      //   elementForward = currentElement.nextElementSibling.nextElementSibling;
+      // }
+      // eslint-disable-next-line max-len
+      elementForward = (page === 0) ? currentElement.nextElementSibling.nextElementSibling.nextElementSibling : currentElement.nextElementSibling.nextElementSibling;
```

Comment:
Nopes, it will not be, since these are under the condition if if (totalPages > paginationLimit) . Here Pagination Limit is 5. If totalPages is more than 5 then only, rest of the logic will apply. Please advise.